modules/programs/sapling.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.sapling;
+
+  iniFormat = pkgs.formats.ini { };
+
+in
+{
+  meta.maintainers = [ maintainers.pbar ];
+
+  options = {
+    programs.sapling = {
+      enable = mkEnableOption "Sapling";
+
+      package = mkPackageOption pkgs "sapling" { };
+
+      userName = mkOption {
+        type = types.str;
+        description = "Default user name to use.";
+      };
+
+      userEmail = mkOption {
+        type = types.str;
+        description = "Default user email to use.";
+      };
+
+      aliases = mkOption {
+        type = types.attrsOf types.anything;
```

Comment:
Is it really possible to have anything as an alias? I would have guessed that `types.attrsOf types.str` would be sufficient.